_layouts/newsletter.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>

```

Comment:
You could inherit from `default.html` so that you don't need to start from the doctype and duplicate the `<head>`.

Reply:
No. OG properties are different etc.. HTML is a good point though.